tests/Bridge/Symfony/DependencyInjection/Greg0ireEnumExtensionTest.php

```diff
@@ -2,24 +2,27 @@
 
 namespace Greg0ire\Enum\Tests\Bridge\Symfony\DependencyInjection;
 
+use Greg0ire\Enum\Bridge\Symfony\DependencyInjection\Compiler\TranslatorCompilerPass;
 use Greg0ire\Enum\Bridge\Symfony\DependencyInjection\Greg0ireEnumExtension;
 use Greg0ire\Enum\Bridge\Twig\Extension\EnumExtension;
 use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\FrameworkExtension;
+use Symfony\Component\DependencyInjection\Reference;
 
 /**
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
-final class Greg0ireEnumExtensionTest extends AbstractExtensionTestCase
```

Comment:
Why removing the `final` keyword here?

Reply:
Woops forgot to add it back, I extended that class in the compiler test case, and figured out it would be simpler to merge both test cases.